README.md

```diff
@@ -1,16 +1,17 @@
-# Genymotion dev and release keys.
+# Genymotion development and release keys
 
 Copyright (C) Genymobile
 
 ## Description
 
-Repository to store dev and release keys, and corresponding .mk that are included from device.mk.
+This repository stores Genymotion Android development and release keys, and corresponding .mk that are included from device.mk.
 
-dev-keys/ is used in device/genymotion/vbox86/device.mk to sign the system at build (properties are flagged with dev-keys).
-We use dev-keys/testkey, but the build system flag for dev-keys, as they are not the AOSP default keys.
-See : https://android.googlesource.com/platform/build/+/refs/tags/android-10.0.0_r25/core/Makefile#306
+`dev-keys` are used in `device.mk` to sign the system at build (properties are flagged with dev-keys).
+We use `dev-keys/testkey`, but the build system flags them as `dev-keys`, as they are not the default AOSP keys.
 
-release-keys/ is used with signature script to resign as release (properties are flagged with release-keys, apk are resigned).
+See: https://android.googlesource.com/platform/build/+/refs/tags/android-10.0.0_r25/core/Makefile#306
+
+`release-keys` are used by signature script to re-sign the build as a release one. Properties are flagged as `release-keys`, apk files are re-signed.
 
 ## Key generation
```

Comment:
Do we need to keep this as users are not building Genymotion images?

Reply:
Maybe I can add something like: "internal use"